tests/unit/features/partitions/test_lifecycle_manager.py

```diff
@@ -154,7 +205,53 @@ def test_partitions_invalid(self, new_dir, parts_data, partitions):
         """Raise an error if partitions are not lowercase alphabetical characters."""
         with pytest.raises(
             errors.FeatureError,
-            match=r"Partition '[\w-]*' must only contain lowercase letters.*",
+            match=(
+                r"Partition '[\w-]*' is invalid.\n"
+                r"Partitions must only contain lowercase letters.*"
+            ),
+        ):
+            lifecycle_manager.LifecycleManager(
+                parts_data,
+                application_name="test_manager",
+                cache_dir=new_dir,
+                partitions=partitions,
+            )
+
+    @pytest.mark.parametrize(
+        "partitions",
+        [
+            ["default", "/"],
+            ["default", "/a"],
+            ["default", "a/"],
+            ["default", "a/b/"],
+            ["default", "/a/b"],
+            ["default", "a//b"],
+            ["default", "/a/"],
+            ["default", "/-"],
+            ["default", "-/"],
+            ["default", "-/-"],
+            ["default", "a/-"],
+            ["default", "-/b"],
+            ["default", "a/-b"],
+            ["default", "a/b-"],
+            ["default", "a/-b-"],
+            ["default", "-a/b"],
+            ["default", "a-/b"],
+            ["default", "-a-/b"],
+            ["default", "a/Test"],
+            ["default", "a/TEST"],
+            ["default", "a/test1"],
+            ["default", "Test/a"],
+            ["default", "TEST/a"],
+            ["default", "test1/a"],
+            ["default", "te-st/a"],
```

Comment:
Just to cover all of our bases:

```suggestion
            ["default", "te-st/a"],
            pytest.param(["default", "test/ο"], id="uses-omicron"),
```

Reply:
Nice!